docs/write-side.md

```diff
@@ -74,53 +74,22 @@ You can emit aggregate commands in the following cases:
 
```

Comment:
Please update the following lines above:

Line 12:
`Since the write side is used only to perform commands, your aggregate can be compact, and only keep state required for command execution.`

Please change "keep state required" to "keep the state required".

Line 14:
`See Martin Fowler's definition for aggregates in the DDD paradigm: [https://martinfowler.com/bliki/DDD_Aggregate.html](https://martinfowler.com/bliki/DDD_Aggregate.html)`

Please add a period at the end of the sentence.

Line 49:
`To configure aggregates in a reSolve app, provide an aggregates array in the application configuration file:`

Please rewrite without using "provide".

Reply:
Done 
>Please rewrite without using "provide".
Changed to "specify"